### PR TITLE
Hw1

### PR DESCRIPTION
**Pull request by sashetov**
Have run through eslint for airbnb style.
Have tested the script and it looks correct.

### CODE REVIEW

**Comment by sashetov** [comment]
@prachi-kashyap @JeremyIV @imOnVacation the PR is ready, could use some reviews

**Review by imOnVacation — COMMENTED** [review]
I like you declared a valuable (thingToSay) and store value i in it.
Your code style gave me some idea of how to approach of Airbnb style, thank you for that!
I like it overall and good job! :)

**Review by prachi-kashyap — COMMENTED** [review]
The code seems perfectly fine to me. You have also followed the best practices and its nicely done.

**Review by prachi-kashyap — APPROVED** [review]

File: hw1/01-fizzbuzz.js [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
 const fizzbuzz = () => {
-  // Add your code here
+  for (let i = 1; i <= 100; i += 1) {
+    let thingToSay = i;
+    if (i % 5 === 0 && i % 3 === 0) {
+      thingToSay = 'FizzBuzz!';
+    } else if (i % 3 === 0) {
+      thingToSay = 'Fizz!';
+    } else if (i % 5 === 0) {
+      thingToSay = 'Buzz!';
```
Comment (vuong2-pdx) [inline hw1/01-fizzbuzz.js:20]: hi I'm late to the party since you merged but just wanted to say good job! I like the quotation marks.

**Pull request merged**